features/logging/cluster_logging_operator.feature

```diff
@@ -152,3 +152,63 @@ Feature: cluster-logging-operator related test
       | card_name | #{cb.card} |
     Then the step should succeed
     """
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-33868
+  @admin
+  @destructive
+  Scenario: Expose more fluentd knobs to support optimizing fluentd for different environments - Invalid Values
+    Given I obtain test data file "logging/clusterlogging/cl_fluentd-buffer_Invalid.yaml"
+    When I run the :create client command with:
```

Comment:
@gkarager does step `I create clusterlogging instance with` work? I was thinking if we should add clean-up steps to remove cl/instance when there has bug and the creation succeeds.

Reply:
@QiaolingTang  The step `I create clusterlogging instance with` throws `Unable to create clusterlogging instance (RuntimeError) ` error. 
Added an extra step to clean up if cl instance created at all.
`And I delete the clusterlogging instance`